### PR TITLE
ClockSpeed not defined in /proc/cpuinfo for 32 bit arm, so return 0.

### PR DESCRIPTION
**Pull request by ausil**
Follow the same behaviour as aarh64 and s390/x

Signed-off-by: Dennis Gilmore <dennis@ausil.us>

### CODE REVIEW

File: utils/machine/machine.go [hunk 1]
```diff
@@ -49,8 +49,8 @@ const maxFreqFile = "/sys/devices/system/cpu/cpu0/cpufreq/cpuinfo_max_freq"
 
 // GetClockSpeed returns the CPU clock speed, given a []byte formatted as the /proc/cpuinfo file.
 func GetClockSpeed(procInfo []byte) (uint64, error) {
-	// s390/s390x and aarch64 changes
-	if true == isSystemZ() || true == isAArch64() {
+	// s390/s390x, aarch64 and arm32 changes
+	if true == isSystemZ() || true == isAArch64() || true == isArm32() {
```
Comment (timstclair) [inline utils/machine/machine.go]: Drop the `true == ` everywhere. Same below.

File: utils/machine/machine.go [hunk 2]
```diff
@@ -280,6 +280,17 @@ func getMachineArch() (string, error) {
 	return arch, nil
 }
 
+// arm32 chanes
+func isArm32() bool {
+        arch, err := getMachineArch()
+        if err == nil {
+        i        if true == strings.Contains(arch, "arm") {
```
Comment (timstclair) [inline utils/machine/machine.go]: stray `i`

Comment (timstclair) [inline utils/machine/machine.go]: Just do `return strings.Contains(arch, "arm")`

**Comment by ausil** [comment]
Thanks for the comments, pushed up updated patch without the extra i and a fix for the feedback

**Comment by timstclair** [comment]
Looks good, please resolve the CLA issues.

**Comment by ausil** [comment]
I signed it!

**Comment by timstclair** [comment]
ok to test

**Comment by luxas** [comment]
Please squash to one commit, then LGTM for me. 
@vishh

**Comment by ausil** [comment]
commits squashed into 1

**Comment by luxas** [comment]
You have to `gofmt -s -w utils/machine/machine.go`

**Comment by timstclair** [comment]
LGTM

**Pull request merged**